packages/components/progress-bar/src/progress-bar.scss

```diff
@@ -0,0 +1,109 @@
+$states: active, success, warning, error;
+
+:host {
+  --_border-radius: var(--sl-border-radius-full);
+  --_label-font: var(--sl-text-input-field-label-label-md);
+  --_helper-font: var(--sl-text-input-field-label-hint-md);
+  --_helper-color: var(--sl-color-input-helper-text-default);
+  --_helper-color-error: var(--sl-color-progressbar-error-track);
+  --_height: var(--sl-size-progressbar);
```

Comment:
Since you already have `--_bar-background`, rename this to `--_bar-thickness` perhaps?